src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.SortCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new SortCommand object.
+ */
+public class SortCommandParser implements Parser<SortCommand> {
+
+    private static final String SORT_BY_NAME = "name";
```

Comment:
as mentioned, might be able to utilise a separate enum class to reduce usage of variable memory like this and in the conditionals below